scripts/install/usr/libexec/cortx-motr/motr-free-space-monitor

```diff
@@ -182,11 +190,11 @@ if __name__ == '__main__':
 
         cluster_usage_percentage = cluster_size[USAGE_PERCENTAGE]
         if cluster_usage_percentage > iem_min_threshold:
```

Comment:
Considering the threshold example of : 70 80 85 90.
I see that if the storage is getting filled and exceeds 70% then we will send one alert and another when we cross over 80%. But at this point if the storage space is freed by doing deletes and the storage reaches 77% and again if the storage starts to fill (because of Object writes) and exceeds 80% then we will not be raising any alerts. Is that by design?
I  see that if the storage occupancy drops below 70% and then starts to increase again in which case we will be having an alert when we exceed the minimum threshold. Just confirm what I mention is by design and then that is fine. Or maybe you might want to ask Pradeep or someone to know the times when the alert needs to be triggered.

Reply:
I am not sure weather it is by design or not need to be confirmed with someone.

Reply:
if the storage size drops less than minimum threshold then only the alert would be sent, if they delete few things then CSM dashboard has a dial pad which shows the current usage.